docs/docs/quickstart.md

```diff
@@ -0,0 +1,94 @@
+---
+sidebar_position: 1.1
+---
+
+# Quick Start
+
+## 1. Run the docker script
+
+```
+docker run -d -t -i \
+-e KLAW_CLUSTERAPI_ACCESS_BASE64_SECRET="dGhpcyBpcyBhIHNlY3JldCB0byBhY2Nlc3MgY2x1c3RlcmFwaQ==" \
+-p 9343:9343 \
+--name klaw-cluster-api aivenoy/klaw-cluster-api:nightly --add-host host.docker.internal:host-gateway \
+&& docker run -d -t -i \
+-e KLAW_CLUSTERAPI_ACCESS_BASE64_SECRET="dGhpcyBpcyBhIHNlY3JldCB0byBhY2Nlc3MgY2x1c3RlcmFwaQ==" \
+-e KLAW_QUICKSTART_ENABLED=true \
+-e SPRING_DATASOURCE_URL="jdbc:h2:file:/klaw/klawprodb;DB_CLOSE_ON_EXIT=FALSE;DB_CLOSE_DELAY=-1;MODE=MySQL;CASE_INSENSITIVE_IDENTIFIERS=TRUE;" \
+-p 9097:9097 --name klaw-core aivenoy/klaw-core:nightly \
+&& docker run -d -t -i -p 2181:2181 --add-host host.docker.internal:host-gateway \
+-e ALLOW_ANONYMOUS_LOGIN=yes --name klaw-zookeeper bitnami/zookeeper:3.8 \
+&& docker run -d -t -i -p 9092:9092 --add-host host.docker.internal:host-gateway \
+-e KAFKA_CFG_ZOOKEEPER_CONNECT=host.docker.internal:2181 -e ALLOW_PLAINTEXT_LISTENER=yes \
+-e KAFKA_ADVERTISED_LISTENERS=PLAINTEXT://host.docker.internal:9092 \
+--name klaw-kafka bitnami/kafka:3.3 \
+&& docker run -d -t -i -p 8081:8081 --add-host host.docker.internal:host-gateway \
+-e SCHEMA_REGISTRY_KAFKA_BROKERS=PLAINTEXT://host.docker.internal:9092 \
+--name klaw-schema-registry bitnami/schema-registry:latest
+```
+
+This script basically creates klaw, kafka and schema registry processes, and also configures kafka and schema registry servers in klaw.
+
+## 2. Login to Klaw
+
+After docker has started 5 processes, you may verify them by `docker ps`.
+Now you may login to Klaw here  [http://localhost:9097/](http://localhost:9097/) with credentials `william : welcome`
+
+![image](../static/images/quickstart/QS_LoginScreen.png)
+
+## 3. Topics Catalog page
+
+After logging in, you will be redirected to a UI. 
```

Comment:
After logging in you will be redirected to our brand new feature rich React UI.

Reply:
You can "Request new Topic" from the top right corner, or you can select the drop down "Request a new" to view all your options.